src/main/groovy/life/qbic/datamodel/dtos/business/ProductItem.groovy

```diff
@@ -23,10 +23,39 @@ class ProductItem {
      */
     final Product product
 
+    /**
+     * Describes the total price of an item based on the quantity and unit price of a product
+     */
+    final double totalPrice
+
+    /**
+     * Describe the total discount price for a product (based on the quantity)
+     */
+    final double totalDiscountPrice
```

Comment:
The commercial term is `quantity discount`, and should be the financial volume difference to the total price.
```suggestion
    final double quantityDiscount
```

Reply:
Can we do quanityDiscountPrice? So its clear its a price and not the ratio? @sven1103